cp-styles/app/styles/components/_checkboxes-and-radios.scss

```diff
@@ -466,3 +466,17 @@ div[data-input-value="category_fields"] {
 		}
 	}
 }
+
+// div:not('[data-input-value^="categories"]') {
+// 	.checkbox-label {
+// 		pointer-events: none;
+
+// 		input {
+// 			pointer-events: auto;
+// 		}
+
+// 		.checkbox-label__text {
+// 			pointer-events: auto;
+// 		}
+// 	}
+// }
```

Comment:
@Yulyaswan should this be removed completely?

Reply:
exactly. removed